WebViewControl/HttpResourceHandler.cs

```diff
@@ -27,10 +28,14 @@ internal class HttpResourceHandler : DefaultResourceHandler {
                         httpRequest.Headers.Add(key, headers[key]);
                     }
 
-                    var response = await httpRequest.GetResponseAsync();
+                    var response = (HttpWebResponse) await httpRequest.GetResponseAsync();
                     Response = response.GetResponseStream();
                     Headers = response.Headers;
 
+                    this.MimeType = response.Headers[ContentTypeHeaderKey];
```

Comment:
response.ContentType
```suggestion
                   MimeType = response.ContentType;
```